liblll/CompilerState.cpp

```diff
@@ -82,5 +82,5 @@ void CompilerState::populateStandard()
 	"(def 'shl (val shift) (mul val (exp 2 shift)))"
 	"(def 'shr (val shift) (div val (exp 2 shift)))"
 	"}";
-	CodeFragment::compile(s, *this, CodeFragment::ReadCallback());
+	CodeFragment::compile(std::move(s), *this, CodeFragment::ReadCallback());
```

Comment:
``s`` is static, so there is only one global instance that will be shared by every call to this function - hence you shouldn't move this, since if you do, the second call to this function will have an empty ``s``. That is, if ``s`` weren't ``const`` anyways, which makes the ``std::move`` a no-operation - in any case, this should be removed.